.mlc_config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "aliveStatusCodes": [999,200,403]
```

Comment:
is 999 and 403 required? I thought this was specific to another repo case pointing to LinkedIN URLs.

Reply:
Done